SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -235,10 +235,11 @@ public SingularityRequestParent bounce(String requestId, Optional<SingularityBou
     final String deployId = getAndCheckDeployId(requestId);
 
     checkConflict(!(requestManager.markAsBouncing(requestId) == SingularityCreateResult.EXISTED), "%s is already bouncing", requestId);
+    Optional<Boolean> removeFromLoadBalancer = Optional.absent();
```

Comment:
can probably just put `Optional.absent()` in the args rather than setting it here